v0/destinations/fb_custom_audience/transform.js

```diff
@@ -155,29 +155,35 @@ const prepareDataField = (
   userSchema,
   userUpdateList,
   isHashRequired,
-  disableFormat
+  disableFormat,
+  skipVerify
 ) => {
   const data = [];
   let updatedProperty;
   let dataElement;
   userUpdateList.forEach(eachUser => {
     dataElement = [];
     userSchema.forEach(eachProperty => {
-      if (isDefinedAndNotNull(eachUser[eachProperty])) {
+      // if skip verify is true we replace undefined/null user properties with empty string
+      let userProperty = eachUser[eachProperty];
+      if (skipVerify && !isDefinedAndNotNull(userProperty)) {
```

Comment:
If the destination doesn't work when we send `null` or `undefined` properties, why not make it the default behavior instead of controlling it with a flag (`skipVerify`)?

Reply:
1. To have backward compatibility to some extend
2. To avoid silent failures for customers who don't want to enable `skipVerify`

Reply:
There is no need to be backward compatible here as the previous version is incorrect anyway.

Reply:
Ok then making it a default behviour

Reply:
As per discussion we are keeping this feature for customers wishing to avoid silent failures